.snapcraft/resources/prepareRocketChat

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-curl -SLf "https://releases.rocket.chat/#{RC_VERSION}/download/" -o rocket.chat.tgz
+curl -SLf "https://releases.rocket.chat/2.1.0/download/" -o rocket.chat.tgz
```

Comment:
```suggestion
curl -SLf "https://releases.rocket.chat/#{RC_VERSION}/download/" -o rocket.chat.tgz
```
Before we merge we'll need to make sure to change this back.  Can leave for now while testing